src/caffe/util/blocking_queue.cpp

```diff
@@ -88,7 +88,8 @@ size_t BlockingQueue<T>::size() const {
 
 template class BlockingQueue<Batch<float>*>;
 template class BlockingQueue<Batch<double>*>;
-template class BlockingQueue<Datum*>;
+// template class BlockingQueue<Datum*>;
```

Comment:
Agreed, I forgot to remove it completely after checking it wasn't needed for compilation